.github/workflows/docker.yml

```diff
@@ -8,10 +8,13 @@ on:
 
 env:
   TEST_TAG: corefiling/pdf2html:test
+  CI_TAG: corefiling/pdf2html:ci-${{ github.run_id }}-${{ github.run_number }}
+  TAG_TAG: corefiling/pdf2html:${{ github.ref_name }}
 
 jobs:
   docker:
     runs-on: ubuntu-latest
+    environment: "PDF2HTML Public Upload Env"
```

Comment:
The secrets used to login into Docker are specified in that environment.
Feels like there should be a way to have secrets not tied to a specific environment but that's the way swhite set it up for now.

Reply:
Yep, let's leave it like this for now then.